marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/entitySearch/GetRecordTest.java

```diff
@@ -0,0 +1,181 @@
+package com.marklogic.hub.dataservices.entitySearch;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.Format;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.dataservices.EntitySearchService;
+import com.marklogic.hub.flow.FlowInputs;
+import com.marklogic.hub.flow.FlowRunner;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.flow.impl.FlowRunnerImpl;
+import com.marklogic.hub.test.Customer;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+//This test is done at the Java layer as multiple scenarios cannot be tested in ml-unit-tests due to the optic calls occurring in the same request.
+// There appears to be a bug with Optic caching values in the same request that is being investigated.
+public class GetRecordTest extends AbstractHubCoreTest {
+
+    private EntitySearchService service;
+    private ReferenceModelProject project;
+
+    @BeforeEach
+    void beforeEach() {
+        service = EntitySearchService.on(getHubClient().getFinalClient());
+        project = installReferenceModelProject();
+    }
+
+    @Test
+    void testGetRecord() {
```

Comment:
To make it clear what this scenario is testing, name this "recordHasMetadataButNoProvenance".

Reply:
This is testing all other properties in the response from getRecord endpoint except for the history. There are 3 methods for 3 different scenarios for record history